test/parallel/test-fs-read-buffer-zero-length.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+var common = require('../common');
+var assert = require('assert');
+var path = require('path'),
```

Comment:
As far as I know, the eslint rule only catches `const` vs `let`, not vs `var`..

cc @silverwind?